examples/duet/vae/AutoEncoder_Syft_Data_Scientist.ipynb

```diff
@@ -1,8 +1,395 @@
 {
```

Comment:
<p>Could we display the last created image in the notebook and have it commented out so it doesnt happen in CI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Done